openedx/core/djangoapps/user_api/config/waffle.py

```diff
@@ -5,17 +5,18 @@
 
 from django.utils.translation import ugettext_lazy as _
 
-from edx_toggles.toggles import LegacyWaffleSwitchNamespace
+from edx_toggles.toggles import WaffleSwitch
 
 SYSTEM_MAINTENANCE_MSG = _(u'System maintenance in progress. Please try again later.')
-WAFFLE_NAMESPACE = u'user_api'
 
-# Switches
-ENABLE_MULTIPLE_USER_ENTERPRISES_FEATURE = u'enable_multiple_user_enterprises_feature'
-
-
-def waffle():
-    """
-    Returns the namespaced, cached, audited Waffle class for user_api.
-    """
-    return LegacyWaffleSwitchNamespace(name=WAFFLE_NAMESPACE, log_prefix=u'UserAPI: ')
+# .. toggle_name: user_api.enable_multiple_user_enterprises_feature
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: If enabled than learners linked to multiple enterprises will be asked to select default
+#   enterprise for current session."
+# .. toggle_use_cases: Permanent
```

Comment:
Not a valid use case. See https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html